concepts/function-arguments/about.md

```diff
@@ -1,2 +1,214 @@
-#TODO: Add about for this concept.
+# About
 
+For the basics on function arguments, please see the [function concept][function concept].
+
+## Parameter Names
+
+Paramater names, like variable names, must start with a letter or underscore and may contain letters, underscores, or numbers.
+Parameter names should not contain spaces or punctuation.
+
+## Positional Arguments
+
+Positional arguments are for parameters defined without a [default argument][default arguments].
+Positional arguments can optionally be called by their name.
+
+Following is an example of positional arguments being called by position and by their name:
+
+```python
+>>> def concat(greeting, name):
+...         return f"{greeting}{name}"
+... 
+>>> print(concat("Hello, ", "Bob"))
+Hello, Bob
+>>> print(concat(name="Bob", greeting="Hello, "))
+Hello, Bob
+
+```
+
+The first call to `concat` passes the arguments by position.
+The second call to `concat` passes the arguments by name, allowing their positions to be changed.
+
+Note that positional arguments cannot follow keyword arguments.

```

Comment:
In a sense, underneath they're all keyword arguments.  But when assigning _default values_ during function definition, you do have to have the default value assignment follow any parameters that _don't_ have default values....so there is a parallel with when a function is called.  Effectively, Python can't tell the difference between the keyword vs positional _unless_ positional always comes first in both calling and in defining functions.

Reply:
The reason I put this here is to try to separate calling from defining, because default arguments are for definition, and keyword arguments are for calling...but as long as we are super-clear in our examples, we don't have to bludgeon people with it here.